### PR TITLE
Added drop records table migration

### PR DESCRIPTION
**Pull request by sahalliburton**
### Description
Added rails migration to remove the `records` table as it has been renamed to `records2020`

See [this document](https://hackmd.io/KOW31Pi1SGWZarqMsh2nfA?view#Resolution) for further details

### CODE REVIEW

**Review by aroltsch — CHANGES_REQUESTED** [review]
I think you'd also need to run db:migrate locally to update schema.rb

**Comment by sahalliburton** [comment]
> I think you'd also need to run db:migrate locally to update schema.rb

Apologies, it did not commit the first time evidently. I have just pushed the update, thanks for catching that!

**Review by alisan16 — APPROVED** [review]
LGTM, please figure out the failure in the rake task before merging

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ source ENV["GEM_SERVER_URL"] || "https://rubygems.org"
 gem "active_model_serializers"
 gem "activejob_dj_overrides"
 gem "aws-sdk", "~> 2"
-gem "bgs", git: "https://github.com/department-of-veterans-affairs/ruby-bgs.git", ref: "8c54411f5a34844dc378e5aa0995730636a505a8"
+gem "bgs", git: "https://github.com/department-of-veterans-affairs/ruby-bgs.git", ref: "7d7c67f7bad5e5aa03e257f0d8e57a4aa1a6cbbf"
```
Comment (sahalliburton) [inline Gemfile:8]: Updated per instructions form [this slack thread](https://dsva.slack.com/archives/C2ZAMLK88/p1609807626159400)

File: .security.yml [hunk 2]
```diff
@@ -1,3 +1,2 @@
 CVES:
-  CVE-2015-9284: 2020-12-31
-  CVE-2017-1002201: 2020-12-31
+  CVE-2015-9284: 2021-06-30
```
Comment (sahalliburton) [inline .security.yml:2]: Moved the audit date out another 6 months as there is still no patch and removed an old security audit that seems to have been resolved

**Review by aroltsch — APPROVED** [review]
LGTM

**Pull request merged**